### DISCUSSION TITLE
LineageOS 20 with MicroG

### DISCUSSION THREAD

**Discussion by delmocro** [agronick/aa-torque#69] [Q&A]
https://github.com/agronick/aa-torque/assets/36604624/5c20fc47-c630-4e48-8436-fe9b06990407

Hello, thanks for making Torque available on AA. Ik have installed everything the right way and AA Torque shows up as on the headunit. The problem i have is that the AA Torque application on the phone is unresponsive. None of the buttons react to input. I tried doing this multiple times while also having Torque pro working in the background. Just doesnt respond to input.  I also uninstalled and installed multiple times just cant get is to respond. Does this have anything  to do with MicroG?

**Comment by agronick** [comment]
It looks like they have an emulator but it is a pain to get working. If you can find binaries of the emulator for linux I would test it. Otherwise you'll need to hook up a debugger yourself.
